Test/ivar_atomic.m

```diff
@@ -0,0 +1,24 @@
+#include "Test.h"
+
+#import <stdatomic.h>
+
+@interface Dummy : Test
+{
+	atomic_bool atomicBool;
+}
+@end
+
+@implementation Dummy
+- (void)test
+{
+  int value = 1;
+  object_setIvar(self, class_getInstanceVariable(object_getClass(self), "atomicBool"), (__bridge id)(void*)(intptr_t)value);
```

Comment:
This is interesting.  It's probably fine to do a non-atomic store of this, but if you do `_Atomic(struct foo)`, where `struct foo` is a large value, then we should be calling into the relevant libatomic functions for atomic operations to make sure that we acquire the correct locks.  I doubt Apple's runtime does this correctly, but we should probably open an issue to track it.

Reply:
Yeah I realize that this doesn’t provide atomicity, it was merely meant as a test for the ivar bug (strictly speaking just compiling the class would have been enough to test for #101). Let me know if I should change the test in any way.